app/components/Play/Play.js

```diff
@@ -1,32 +1,37 @@
 import React, { Component } from 'react';
-import { PickEmotion } from '../PickEmotion/PickEmotion';
+import PickEmotion from '../PickEmotion/PickEmotion';
 import ImageCapture from '../ImageCapture/ImageCapture';
 import { EmotionResults } from '../EmotionResults/EmotionResults';
+import { analyzeResponse } from '../../helper.js';
 
 export default class Play extends Component {
   constructor() {
     super()
     this.state = {
-      emotions: {}
+      emotions: []
     }
   }
 
-  analyzeEmotions(faceURL) {
+  analyzeEmotions(faceBlob) {
+    const body = faceBlob;
+
+    const headers = new Headers({
+      'Content-Type': 'application/octet-stream',
+      'Ocp-Apim-Subscription-Key': '5044b8a6914442cfbb74003672a494a0'
+    })
+
+
```

Comment:
nah you have it right, my fault.  you have to use `headers.get` to see the keys.  it's working how you have it.